db/warehouse/migrate/20190821200216_add_housing_status_to_hmis_forms.rb

```diff
@@ -0,0 +1,5 @@
+class AddHousingStatusToHmisForms < ActiveRecord::Migration
+  def change
+    add_column :hmis_forms, :housing_status, :string
```

Comment:
We'll need a method to call on HmisForm to pre-populate this for existing form responses.